### PR TITLE
Migrate QnA Maker quickstart sample code

### PR DESCRIPTION
**Pull request by v-jaswel**

### CODE REVIEW

File: go/QnAMaker/rest/create-kb.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    "bytes"
+    "encoding/json"
+    "fmt"
+    "io/ioutil"
+    "net/http"
+    "strconv"
+    "time"
+)
+
+var host string = "https://{your-resource-name}.api.cognitive.microsoft.com"
```
Comment (sahithikkss) [inline go/QnAMaker/rest/create-kb.go:13]: URL format is currently the following .
https://{your-resource-name}.cognitiveservices.azure.com
Please change across all languages code samples.

File: go/QnAMaker/rest/publish-kb.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+// 2. Compile with: go build publish-kb.go
+// 3. Execute with: ./publish-kb
+// 4. For successful publish, no data is returned, only 204 http status
+
+func main() {
+
+	var knowledge_base_id = "YOUR-KNOWLEDGE-BASE-ID";
+	var resource_key = "YOUR-RESOURCE-KEY";
+
+	var host = fmt.Sprintf("https://YOUR-RESOURCE-NAME.api.cognitive.microsoft.com/qnamaker/v4.0/knowledgebases/%s", knowledge_base_id);
```
Comment (sahithikkss) [inline go/QnAMaker/rest/publish-kb.go:19]: Cognitive Service endpoint - change here too

**Review by sahithikkss — CHANGES_REQUESTED** [review]
Left a comment on string change for cognitive resource name.

Optional: 
Will it be possible to change(unify across languages) the reading of configuration strings (resource key, endpoint key, resource name, endpoint host name, kbid in same manner. It is environment variables in a few languages and sample strings are present as literals in a few other languages.

**Comment by v-jaswel** [comment]
I will incorporate the changes described in sahithikkss's comments in a second PR.

**Pull request merged**